app/models/user.rb

```diff
@@ -26,13 +26,22 @@ def init_role
     self.role ||= :standard
   end
 
+  protected
+  def confirmation_required?
+    false
+  end
+
   private
 
-  def downgrade
-    self.role = "standard"
-    wikis.each do |wiki|
-      wiki.make_public
-    end
-    save
-  end
+  #def send_confirmation_email
```

Comment:
I would remove the commented  out code, unless you are keeping it around for any reason!